cmd/snap/cmd_debug_snap_downloads_cache.go

```diff
@@ -0,0 +1,87 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/store"
+	"github.com/snapcore/snapd/strutil/quantity"
+)
+
+const longSnapsCacheHelp = `
+Show statistics of the local snap downloads cache.
+`
+
+type cmdSnapDownloadsCache struct {
+	Dir      string `long:"cache"`
+	MaxItems int    `long:"max-items"`
+}
+
+func init() {
+	addDebugCommand("snap-downloads-cache",
+		"Show statistics of the local snaps download cache",
+		longSnapsCacheHelp,
+		func() flags.Commander {
+			return &cmdSnapDownloadsCache{}
+		}, map[string]string{
+			"cache":     "Cache directory, if different than the default location",
+			"max-items": "Maximum count of cache-unique items, if different than the default",
```

Comment:
Why do we need this? It has no effect on `Stats`

Reply:
hm hm, was bit confused why this doesn't look right and yeah I had some changes that I stashed but then forgot to commit.

Reply:
Thanks for the update. Is it worth adding a test with an output sample? I see the test coverage for debug commands is fairly hit and miss so it's up to you